fetch_new_accessions.bash

```diff
@@ -45,13 +45,14 @@ while read -r taxon_id; do
     formatted_query="$(python3 ../../format_query.py "$taxon_id")"
     formatted_data="result=READ_STUDY&query=${formatted_query}&display=xml"
 
-    if $UPDATE || [ ! -e data.xml ]; then
-    	curl 'https://www.ebi.ac.uk/ena/data/warehouse/search' --data "$formatted_data" --compressed > data.xml
+    echo $formatted_data
```

Comment:
Is `$formatted_data` something we would always want, or is this a debugging `echo`?